app/main.py

```diff
@@ -494,9 +494,22 @@ def initialize(self, request, response):
 
     def serve(self):
         request, response, env = self.request, self.response, self.env
+
+        # If the Person Finder instance has not been initialized yet,
+        # prepend to any served page a warning and a link to initialize
+        # the datastore.
+        if 0 == len(env.repo_options) and env.action != 'setup_datastore':
```

Comment:
len(env.repo_options) is not a good way to check if the instance is initialized. env.repo_options only returns active repositories, so it can often be empty even in production.

Can you define a new config, named "initialized" or something, which indicates that the instance is initialized? To set the config, you can add it to here:
https://github.com/google/personfinder/blob/master/tools/setup_pf.py#L62
To read the config, you can write config.get("initialized").